src/main/java/com/greenfox/poker/model/Deck.java

```diff
@@ -0,0 +1,19 @@
+package com.greenfox.poker.model;
+
+import java.util.List;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import org.springframework.stereotype.Component;
+
+
+@Component
+public class Deck {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
The `Deck` won't be an Entity, so you should't use these Entity related annotations here.